### ISSUE TITLE
Enable multiple pools

### ISSUE THREAD

**Issue by pi0neerpat** [open-dollar/od-sdk#53]
Depends on #45
 
Now that we can load data from a single pool, we need to add the ability to pull for multiple pools.

- Rename `fetchNitroPoolODGwstETH` to `fetchNitroPool` which is more generic
- The arguments should accept the token symbol (eg. WSTETH), instead of the address of the pool
- Validate that we are fetching all data needed (see both app screenshots below) 

![image](https://github.com/open-dollar/od-app/assets/35622595/b7e9fc66-59a0-4cba-b9dc-e14a896bf89e)

![image](https://github.com/open-dollar/od-app/assets/35622595/4c5a92c7-ba06-49e8-85e9-fd048b2f747e)

**Comment by jahabeebs** [comment]
Closing as won't do -- part of TVL project
